src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupBlackListReadFilter.java

```diff
@@ -39,6 +36,11 @@
     // Command line parser requires a no-arg constructor
     public ReadGroupBlackListReadFilter() {};
 
+    @Override
+    public void setHeader(final SAMFileHeader samFileHeader) {
+        super.setHeader(samFileHeader);
+        parseReadGroupFilters();
+    }
```

Comment:
comment below says that list files are loaded recursively.  is that true in the standard list file loading which is used now?

Reply:
This shouldn't mention the file expansion at all anymore, so it should be removed. Having said that, I thought it was recursive, but it looks like it isn't.